### ISSUE TITLE
Can not import jsx file to place a checkbox in the headings.

### ISSUE THREAD

**Issue by SimplyCorey** [matfish2/vue-tables-2#66]
- Vue.js version: 2.1.10
- consumed using: webpack

I am trying to import a JSX file into a vuejs component using the import command.  I am getting an error of `Uncaught TypeError: h is not a function.`  I want to add a checkbox to the header of the table but I am unable to figure out a way to do this so far.  I have reviewed issues: #53 and #23 but I still am unable to figure it out.  I am sure that I am doing something wrong, but I am at a loss at the moment.  Any help would be appreciated !

- FULL error message (including stack trace):

```
Uncaught TypeError: h is not a function
    at Object.exports.default [as checkbox] (eval at <anonymous> 
```

- relevant code:

checkbox.jsx:

```
export default function(h) {
    return <h6><input type="checkbox"/>Title</h6>
}

```
Table.vue

```
<template>
    <v-server-table url="/data" :options="options" :columns="columns"></v-server-table>
</template>

<script>
    import checkbox from './templates/checkbox.jsx';

    export default{
        data: function() {
            return {
                columns: ['id', 'name', 'description', 'type', 'actions', 'checkbox'],
                options: {
                    filterByColumn: true,
                    headings: {
                        checkbox
                    }
                }
            }
        }
    }
</script>
```

**Comment by JoshZA** [comment]
This works for me with a client-table, haven't tried with server table.

Have you included the the jsx compiler in your webpack config?

**Comment by awmottaz** [comment]
I am having the same error, except only on second rendering (I have my table in one of two [dynamic components](https://vuejs.org/v2/guide/components.html#Dynamic-Components)). The first time the table is rendered, it works fine. But when switching to the other component and back again, I get the error.

**foo.jsx**

```
export default function(h) { return <span class="foo">Foo</span> }
```

**Fizz.vue**

```
<template>
  <v-client-table :data="tableData" :columns="columns" :options="options"></v-client-table>
</template>

<script>
import foo from './template/foo.jsx'

options{
  headings: {
    foo
  }
}
```

**App.vue**

```
<template>
  <component :is="activeComponent"></component>
  <button @click="toggleActiveComponent"></button>
</template>

<script>
import Fizz from './components/Fizz'
import Buzz from './components/Buzz'

export default {
  data() {
    return { activeComponent: 'fizz' }
  },
  components: {
    'fizz': Fizz,
    'buzz': Buzz
  }
}
```

**Comment by SimplyCorey** [comment]
I gave up trying to deal with JSX and moved on to another package.  I tried everything within my webpack config and was not able to get it to work.

**Comment by matfish2** [comment]
Fixed (See #78)
